integrationTests/tesGameHelperFunctions.go

```diff
@@ -0,0 +1,259 @@
+package integrationTests
```

Comment:
typo in file name. "tes**t**GameHelper..."

Reply:
done